src/styles/index.scss

```diff
@@ -1,3 +1,113 @@
+@import '../utils/variables';
+@import './reset.css';
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    width: 200px;
```

Comment:
The width property here is redundant because its parent (.card) already has a width set. This can lead to unexpected behavior, especially if the parent width changes in the future.